articles/cognitive-services/Translator/document-translation/client-sdks.md

```diff
@@ -180,47 +180,47 @@ pip install azure-ai-translation-document==1.0.0
 Create a new Python application in your preferred editor or IDE. Then import the following libraries.
 
 ```python
-    import os
-    from azure.core.credentials import AzureKeyCredential
-    from azure.ai.translation.document import DocumentTranslationClient
+import os
```

Comment:
The rest of the changes are just indentation fixes so that it's easy to copy/paste these snippets.